skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h

```diff
@@ -43,9 +45,39 @@ struct BaseActorValues
   }
 };
 
-inline BaseActorValues GetBaseActorValues(uint32_t baseId,
+inline BaseActorValues GetBaseActorValues(espm::Loader& espm, uint32_t baseId,
```

Comment:
`raceIdOverride` must override `raceId` from espm if `raceIdOverride` != 0.
please autotest this function